src/Stamp/DebounceStamp.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Krak\SymfonyMessengerRedis\Stamp;
+
+use Symfony\Component\Messenger\Stamp\StampInterface;
+
+/**
+ * Apply this stamp to debounce delivery of your message on a transport.
+ */
+final class DebounceStamp implements StampInterface
+{
+    private $id;
+    private $delay;
+
+    /**
+     * @param string $id unique identifier
+     * @param int $delay The delay in milliseconds
+     */
+    public function __construct(string $id, int $delay)
```

Comment:
I think we should switch the order of these params and make the id optional similar to the Unique stamp where if not set, it defaults to the the hash of the message bus.

Reply:
i have updated it accordingly.